kyaml/openapi/openapi.go

```diff
@@ -100,16 +104,23 @@ var precomputedIsNamespaceScoped = map[yaml.TypeMeta]bool{
 	{APIVersion: "rbac.authorization.k8s.io/v1", Kind: "ClusterRoleBinding"}:                     false,
 	{APIVersion: "rbac.authorization.k8s.io/v1", Kind: "Role"}:                                   true,
 	{APIVersion: "rbac.authorization.k8s.io/v1", Kind: "RoleBinding"}:                            true,
+	{APIVersion: "rbac.authorization.k8s.io/v1alpha1", Kind: "ClusterRole"}:                      false,
+	{APIVersion: "rbac.authorization.k8s.io/v1alpha1", Kind: "ClusterRoleBinding"}:               false,
+	{APIVersion: "rbac.authorization.k8s.io/v1alpha1", Kind: "Role"}:                             true,
+	{APIVersion: "rbac.authorization.k8s.io/v1alpha1", Kind: "RoleBinding"}:                      true,
```

Comment:
This is because the JSON schema we had was v1.21.2 and the proto Mengqi added in his benchmark tests was v1.21.8. I didn't see a reason to not keep the bump.

Reply:
Wouldn't that imply that all these alpha kinds were added between those versions? That doesn't seem right, especially since these are such old kinds.  I guess it doesn't hurt to have them, but it makes me a little suspicious of the content. Can we regenerate it, just to be sure that the content wasn't altered during testing?

Reply:
+1
k8s doesn't introduce new APIs between patch versions.
Please double check what's going on

Reply:
I downloaded the v1.21.2 proto schema and the alpha types disappeared from this list, so it seems likely to be that it does come from the version bump.

We can discuss bumping the version separately from this PR. In this PR I decided to just use the v1.21.2 proto schema I fetched.